### ISSUE TITLE
(nerf) doppelganger pilot skill

### ISSUE THREAD

**Issue by korshard** [PavelZinchenko/event-horizon-frontier-builds#366]
apparently the current cp cost of doppelganger frigate pilot is 1 which makes it insanely powercreep with frigate spam strategy especially when used alongside with technician. so i suggest increase the cost to 2 for frigate and make it cannot be used alongside technician skill overall

**Comment by TaipionsSalemClient** [comment]
This topic has been discussed at length already.
For my personal taste, I'd remove the skill entirely, but apparently that is not what people/pavel want...

I'd say it's pretty balanced for destroyers,
Too expensive for cruisers, which is due to how underpowered and weak cruisers are.

It's a clear best-in-slot for frigates, but frigates are not that strong, and there are only a handful of usable frigates anyways,
if you use more than 5 command points for frigates, you're likely doing something wrong.

**Comment by SuplexCity25284** [comment]
Yeah, it makes a cruiser take 5 command points which except for a couple OP ships is plain ridiculous
